components/eventbrite/sources/common-webhook.js

```diff
@@ -0,0 +1,40 @@
+const common = require("./common.js");
+const get = require("lodash/get");
+
+module.exports = {
+  ...common,
+  props: {
+    ...common.props,
+    http: "$.interface.http",
+  },
+  hooks: {
```

Comment:
Make sure you extend the `hooks` from the parent module to avoid future headaches:

```suggestion
  hooks: {
    ...common.hooks,
```

Reply:
Updated